docs/app/Examples/modules/Transition/Explorers/TransitionExampleSingleExplorer.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import { Form, Grid, Image, Transition } from 'semantic-ui-react'
+
+const transitions = [
+  'scale',
+  'fade', 'fade up', 'fade down', 'fade left', 'fade right',
+  'horizontal flip', 'vertical flip',
+  'drop',
+  'fly left', 'fly right', 'fly up', 'fly down',
+  'swing left', 'swing right', 'swing up', 'swing down',
+  'browse', 'browse right',
+  'slide down', 'slide up', 'slide right',
+]
+const options = transitions.map(name => ({ key: name, text: name, value: name }))
+
+export default class TransitionExampleSingleExplorer extends Component {
+  state = { animation: transitions[0], duration: 500, visible: true }
+
+  handleChange = (e, { name, value }) => this.setState({ [name]: value })
+
+  handleVisibility = () => this.setState({ visible: !this.state.visible })
+
+  render() {
+    const { animation, duration, visible } = this.state
+
+    return (
+      <Grid columns={2}>
+        <Grid.Column as={Form}>
+          <Form.Select
+            label='Choose transition'
+            name='animation'
+            onChange={this.handleChange}
+            options={options}
+            value={animation}
+          />
+          <Form.Input
+            label={`Duration: ${duration}ms `}
+            min={100}
+            max={2000}
+            name='duration'
+            onChange={this.handleChange}
+            step={100}
+            type='range'
+            value={duration}
+          />
+          <Form.Button content={visible ? 'Hide' : 'Show'} onClick={this.handleVisibility} />
```

Comment:
When changing state during a transition the example does not behave correctly as the transition is interrupted.  I'd like to disable the form controls while the transition is in progress.

In trying to disable/enable the Show/Hide button in the explorer examples, I now think we need to fire the callbacks on `Transition.Group` for each transition as well.  I'm not sure there is a way to capture start/complete without this.

Would you mind updating the examples to show how this is possible?